pkg/restclient/common.go

```diff
@@ -0,0 +1,73 @@
+package restclient
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+)
+
+const baseURL = "http://localhost:24007"
+
+// RESTClient represents Glusterd2 REST Client
+type RESTClient struct {
+	baseURL  string
+	username string
+	password string
+}
+
+type httpError struct {
+	Error string `json:"Error"`
+}
+
+// NewRESTClient creates new instance of RESTClient
+func NewRESTClient(baseURL string, username string, password string) *RESTClient {
```

Comment:
This can just be `New`, so that users will be calling this as `client.New()` instead of `restclient.NewRESTClient`. Still makes the same amount of sense.